examples/pleading.tex

```diff
@@ -3,7 +3,8 @@
 \casenumber{1}
 \claimant{[CLAIMANT]}
 \defendant{[DEFENDANT]}
-\court{%
+\court%
```

Comment:
`%` after `\court` is unnecessary (i think?)

Reply:
Fixed.